src/common/utxobased/db/makeNewBaselets.ts

```diff
@@ -0,0 +1,189 @@
+import {
```

Comment:
I feel like we should do the same "bridging" of the API change here; that is add/rename any changes and annotate deprecated old code (soon to be dead and removed).